src/core/json-schema-components.jsx

```diff
@@ -335,14 +335,14 @@ export class JsonSchema_boolean extends Component {
     errors = errors.toJS ? errors.toJS() : []
     let enumValue = schema && schema.get ? schema.get("enum") : null
     let allowEmptyValue = !enumValue || !required
-    let booleanValue = !enumValue && fromJS(["true", "false"])
+    let booleanValue = !enumValue && ["true", "false"]
     const Select = getComponent("Select")
 
     return (<Select className={ errors.length ? "invalid" : ""}
                     title={ errors.length ? errors : ""}
                     value={ String(value) }
                     disabled={ disabled }
-                    allowedValues={ enumValue || booleanValue }
+                    allowedValues={ enumValue ? Array(...enumValue) : booleanValue }
```

Comment:
prefer spread array shorthand
```suggestion
                    allowedValues={ enumValue ? [...enumValue] : booleanValue }
```